ci/jenkins/test.sh

```diff
@@ -670,6 +689,150 @@ function clean_tmp() {
     find ${WORKDIR} -name "support-bundles*" -mtime +7 -exec rm -rf {} \; 2>&1 | grep -v "Permission denied" || true
 }
 
+function deliver_k8s() {
+    echo "===== Read k8s node info ====="
+    WORKER_HOSTNAMES=()
+    HOSTNAMES=()
+    WORKER_IPV4S=()
+    IPV4S=()
+    WORKER_IPV6S=()
+    IPV6S=()
+    NODE_HOSTNAMES=`cat ${WORKDIR}/nodes | awk '{print $1}'`
+    while read HOSTNAME; do
+        if [ ${MASTER_HOSTNAME} ]; then
+            WORKER_HOSTNAMES+=("${HOSTNAME}")
+        else
+            MASTER_HOSTNAME=${HOSTNAME}
```

Comment:
Let us start to use CP or ControlPlane, rather than Master.

Reply:
Sure. Fixed.